test/integration/document-file-dependencies/test/index.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import { findPort, killApp, nextBuild, nextStart } from 'next-test-utils'
+import webdriver from 'next-webdriver'
+
+jest.setTimeout(1000 * 60 * 2)
+const appDir = join(__dirname, '..')
+
+let appPort
+let app
+
+describe('File Dependencies', () => {
+  describe('production mode', () => {
+    beforeAll(async () => {
+      appPort = await findPort()
+      await nextBuild(appDir)
+      app = await nextStart(appDir, appPort)
+    })
+
+    afterAll(() => killApp(app))
+
+    it('should apply styles defined in global and module css files in a standard page.', async () => {
+      const browser = await webdriver(appPort, '/')
+      await browser.waitForElementByCss('#index')
+
+      const styles = await browser.eval(() => {
+        const computed = getComputedStyle(document.getElementById('index'))
+        return {
+          color: computed.color,
+          backgroundColor: computed.backgroundColor,
+        }
+      })
+
+      expect(styles).toEqual({
+        color: 'rgb(0, 0, 255)',
+        backgroundColor: 'rgb(200, 200, 200)',
+      })
```

Comment:
@timneutkens Hi, thank you for the review!
I rewrite the integration test and app to check styles applied in webdriver. 😄 
What do you think about this?

Reply:
Great update 💯 this makes it much more reliable looking to the future ✅